docs/docs/dev_docs/contracts/functions.md

```diff
@@ -4,16 +4,20 @@
 
 - A special `constructor` function MUST be declared within a contract's scope.
 - A constructor doesn't have a name, because its purpose is clear: to initialise state.
-- In Aztec terminology, a constructor is always a 'private function' (i.e. it cannot be an `open` function).
+- In Aztec terminology, a constructor is always a 'secret function' (i.e. it cannot be an `open` function).
 - A constructor behaves almost identically to any other function. It's just important for Aztec to be able to identify this function as special: it may only be called once, and will not be deployed as part of the contract.
 
 ## secret functions
 
```

Comment:
```suggestion
> a.k.a. "private" functions
```
(I suggest this, because I'm leaning towards us committing to calling things public and private, longer term. We might end up using annotations to decorate functions, instead of the `open` keyword, in which case `open` will be gone from the language soon (yay!))